ex10/js.js

```diff
@@ -0,0 +1,10 @@
+var fff = _.debounce(function() {
+    if ($( '.input' ).val() === '') {
+      $( "ol" ).addClass( "hide" );
+    } else {  
+      $( "ol" ).removeClass( "hide" );
+    }
+  }, 500) 
+  
+$( '.input' ).on('input', function() { fff(); });
```

Comment:
`$( '.input' ).on('input', function() { fff(); });`
is the same as:
`$( '.input' ).on('input', fff);`